src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.security.SecureRandom;
+
+public class Lottery {
+    private static final int boundRandom = 100;

```

Comment:
constants should be uppercase, 
https://checkstyle.sourceforge.io/reports/google-java-style-20170228.html#s5.2.4-constant-names